docs/data/data-grid/filtering/server-side.md

```diff
@@ -8,6 +8,10 @@ The example below demonstrates how to achieve server-side filtering.
 
 {{"demo": "ServerFilterGrid.js", "bg": "inline"}}
 
+:::info
```

Comment:
I would suggest changing this to a `success` callout, which is a way of saying "here's a tip to improve your implementation of this feature."

```suggestion
:::success
```